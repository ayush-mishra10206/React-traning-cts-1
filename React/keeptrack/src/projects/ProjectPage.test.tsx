import { render } from "@testing-library/react"
import ProjectPage from "./ProjectsPage"
import renderer from 'react-test-renderer';


describe('<ProjectPage/>',()=>{
    beforeEach(()=>{
        render(<ProjectPage/>); 
    })
    xtest('working',()=>{
        console.log('working without render')
        // render(<ProjectPage/>)
    })

    xtest('snapshot',()=>{
        const tree = renderer.create(<ProjectPage/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
});