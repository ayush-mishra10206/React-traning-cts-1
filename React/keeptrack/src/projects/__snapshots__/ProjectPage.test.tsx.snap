// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectPage/> snapshot 1`] = `
Array [
  <h1>
    Projects1
  </h1>,
  <div
    className="row"
  >
    <label
      data-testid="listLabel"
    />
    <div
      className="cols-sm"
    >
      <div
        className="card"
      >
        <label
          data-testid="projectName"
        >
          Something
        </label>
        <img
          alt="Perosn1"
          src="/assets/placeimg_500_300_arch4.jpg"
        />
        <section
          className="section darkgrey"
        >
          <h5
            className="strong"
          >
            <strong>
              Perosn1
            </strong>
          </h5>
          <p>
            Descp1
          </p>
          <p>
            Budget : 
            54,637
          </p>
          <button
            className="bordered"
            data-testid="editBtn"
            onClick={[Function]}
          >
            Edit
          </button>
        </section>
      </div>
    </div>
    <div
      className="cols-sm"
    >
      <div
        className="card"
      >
        <label
          data-testid="projectName"
        >
          Something
        </label>
        <img
          alt="Person2"
          src="/assets/placeimg_500_300_arch1.jpg"
        />
        <section
          className="section darkgrey"
        >
          <h5
            className="strong"
          >
            <strong>
              Person2
            </strong>
          </h5>
          <p>
            Descp2
          </p>
          <p>
            Budget : 
            91,638
          </p>
          <button
            className="bordered"
            data-testid="editBtn"
            onClick={[Function]}
          >
            Edit
          </button>
        </section>
      </div>
    </div>
  </div>,
]
`;
